### DISCUSSION TITLE
Enhancement: Add a new line for "Answer" part of gemini-2.0-flash-thinking-exp

### DISCUSSION THREAD

**Discussion by Originalimoc** [danny-avila/LibreChat#5131] [Feature Requests & Suggestions]
### What features would you like to see added?

Currently the answer and thinking are concatted without even a space. Add a new line would make viewing much better. Even better, consider add "\n## Answer\n" markdown to seperate them.

### Which components are impacted by your request?

UI/API

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Should be directed to the API Provider (Google). I have no control in this, and in my experience, sometimes it does this in the response anyway

**Reply by Originalimoc** [reply]
You actually can do it.
1/ The v1alpha REST API and GenAI SDK( https://ai.google.dev/gemini-api/docs/thinking-mode )(Python though😅) has a parameter stating if current response delta is a thought or not.
2/ The "choices" length is different too, along with v1alpha API:
Example: https://github.com/Originalimoc/OpenAI_API_Adapter-Google/commit/c2dd21d14ef280a27ee40b67b9f21ebe65148e37
